.gitignore

```diff
@@ -1,5 +1,7 @@
 /compile_commands.json
-/build
+/CMakeFiles
+/stagedir
```

Comment:
These shouldn't exist outside the build directory anyway, no need to add them here